bin/mqsub

```diff
@@ -142,6 +282,9 @@ if __name__ == '__main__':
     parser.add_argument('--script-tmpdir', help="When '--script -' is specified, write the script to this location as a temporary file", default='/lustre/scratch/microbiome/tmp')
     parser.add_argument('--poll-interval', help="Poll the PBS server once every this many seconds [default: 30]", type=int, default=30)
     parser.add_argument('--no-executable-check', help="Usually mqsub checks the executable is currently available. Don't do this [default: do check]",action='store_true')
+    parser.add_argument('--file', help="A file with list of newline separated commands to be split into chunks and submitted. One command per line. mqsub --file <file.txt> --chunk-num <int>")
```

Comment:
previous comment applies here still re name of this argument

Reply:
Fair enough. Will update this.